src/common/config_opts.h

```diff
@@ -1050,6 +1050,9 @@ OPTION(bluestore_debug_inject_read_err, OPT_BOOL, false)
 OPTION(bluestore_debug_randomize_serial_transaction, OPT_INT, 0)
 OPTION(bluestore_inject_wal_apply_delay, OPT_FLOAT, 0)
 OPTION(bluestore_shard_finishers, OPT_BOOL, false)
+OPTION(bluestore_rtc, OPT_BOOL, false)
```

Comment:
RTC stands for Run To Completion which means that a thread does work to the end w/o context switch.
In bluestore, RTC means that ShardedOPWQ(thread pool) does WAL commit(nvdimm) w/o context switch. (bluestore_rtc)
Furthermore, ShardedOPWQ does work on behalf of finishers, if possible. (bluestore_no_finisher)
ShardedOPWQ in RTC mode has pg lock already, so i needed to remove a pg lock in callback contexts.

I observed performance increase in no_finisher mode (i7 machine)
2 replicas 2 osds
  bluestore_rtc (12.5KIOPS) -> bluestore_no_finisher (15KIOPS)

Without bluestore_rtc, it's impossible for ShardedOPWQ to do callbacks on behalf of finishers
So i added the patch to remove finisher
If needed, i'll separate the patch into two